### DISCUSSION TITLE
New installation - Internal server error

### DISCUSSION THREAD

**Discussion by ziouz7** [pretix/pretix#2778] [General]
Hi,
I get an "Internal server error" for a Redhat/postgresql/nginx deployment without docker or ssl, using the minimum requirements for evaluation. Pretix and nginx urls are set to "127.0.0.1" on port 80.

pretix-worker.service log:

![image](https://user-images.githubusercontent.com/112698160/188125147-99de92db-746a-401d-90e1-d7b1d3ee0df8.png)

pretix-web.service log:

![image](https://user-images.githubusercontent.com/112698160/188124916-63884637-ebab-4f81-9257-14c7df6c5382.png)

any ideas? Thank you very much in advance.

**Comment by ziouz7** [comment]
Is it the case that celery tries to use RabbitMQ instead of redis? How can I set celerey to use redis and discard RabbitMQ ? 

pretix is installed with the default settings as mentioned in the documentation.

 celery[469412]: [2022-09-22 10:46:28,976: ERROR/MainProcess] consumer: Cannot connect to amqp://guest:**@127.0.0.1:5672//: [Errno 111>

any thoughs?

**Reply by raphaelm** [reply]
This looks like quite some things are off here.

In fact, celery tries to use rabbitmq. That probably means it does not find your configuration file, maybe it is named wrong or in the wrong folder?

the pretix-web esrvice looks like some of the installation steps did not run correctly

**Comment by ziouz7** [comment]
Hello, thanks for your reply. I changed the default configuration to:
/var/pretix/venv/bin/python3 /var/pretix/venv/bin/celery -A pretix.celery_app worker -b redis://guest@127.0.0.1/2 -l info

So now the worker starts-up without issues.

I get multiple '13:Permission denied' errors for ico, css, svg files at the nginx error.log file. I changed the owner and group of "/var/pretix/venv/lib/python3.9/site-packages/pretix/static.dist/;" to nginx:nginx along with chmod 664 for files and chmod 775 for folders. 

Do I need to use www-data:www-data?
Is there another folder(s) to give ownership to nginx user?

**Comment by ziouz7** [comment]
problem solved via: 
#chown -R pretix:pretix /etc/pretix/
